client/test/role.go

```diff
@@ -54,6 +55,10 @@ type (
 		sync.RWMutex
 	}
 
+	BalanceReader interface {
```

Comment:
Its sound pretty self explanatory, but we always comment public stuff in this file.